src/app/modules/group/components/group-join-by-code/group-join-by-code.component.html

```diff
@@ -61,15 +61,15 @@
             #control="ngModel"
             required
           ></alg-duration>
-          <alg-button
+          <p-button
             icon="fa fa-check"
-            class="submit-duration-button"
-            (click)="codeLifetimeControlValue && submitCodeLifetime(codeLifetimeControlValue.ms)"
+            styleClass="alg-button submit-duration-button"
+            (onClick)="codeLifetimeControlValue && submitCodeLifetime(codeLifetimeControlValue.ms)"
             [disabled]="processing || !codeLifetimeControlValue || !control.dirty || codeLifetimeControlValue.ms === group.codeLifetime?.ms"
             [pTooltip]="durationTooltip"
             tooltipPosition="right"
             tooltipEvent="hover"
-          ></alg-button>
+          ></p-button>
```

Comment:
url: https://dev.algorea.org/branch/feature/use-p-button-instead-of-alg-button/en/#/groups/by-id/672913018859223173;path=52767158366271444/details/members
The last button on the right is too high and too wide compared to before
Before:
![image](https://user-images.githubusercontent.com/16257159/168769005-bf2149ce-7d48-4d7f-b4af-ed9cbb43c863.png)

After:
![image](https://user-images.githubusercontent.com/16257159/168770847-b046f0b2-0940-47f7-9bfd-ecbb9033a63c.png)

Reply:
Fixed height for button